website/src/components/Stats/Stats.components.tsx

```diff
@@ -53,6 +53,17 @@ const interestingStates = [
   "halted_by_moderator",
 ];
 
+const stateColors = {
```

Comment:
could you rename to something like `STATE_COLORS`? since this is a constant.

Reply:
Done!